pkg/agent/apiserver/handlers/podinterface/handler.go

```diff
@@ -27,35 +27,35 @@ import (
 
 // Response describes the response struct of pod-interface command.
 type Response struct {
-	PodName       string `json:"name,omitempty" antctl:"name,Name of the Pod"`
-	PodNamespace  string `json:"podNamespace,omitempty"`
-	InterfaceName string `json:"interfaceName,omitempty"`
-	IP            string `json:"ip,omitempty"`
-	MAC           string `json:"mac,omitempty"`
-	PortUUID      string `json:"portUUID,omitempty"`
-	OFPort        int32  `json:"ofPort,omitempty"`
-	ContainerID   string `json:"containerID,omitempty"`
+	PodName       string   `json:"name,omitempty" antctl:"name,Name of the Pod"`
+	PodNamespace  string   `json:"podNamespace,omitempty"`
+	InterfaceName string   `json:"interfaceName,omitempty"`
+	IPs           []string `json:"ips,omitempty"`
```

Comment:
@jianjuns @tnqn any concern about breaking backward-compatibility for that API? No concern on my side since this is an API which is local to the Agent and AFAIK only consumed by antctl.

Reply:
I think it is fine.
But maybe we should think about our API versioning strategy sometime.